Description: update python shebang
Author: Nicolas HOUDELOT <nicolas@demosdebs.org>
Applied-Upstream: no
Last-Update: 2020-04-30
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/run_demo.py
+++ b/run_demo.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python2
 
 """This is a simple pyGtk launcher for the demo, so that you don't need to 
 deal with the command line parameters if you don't want to
